Microsoft.HealthVault/Clients/Deserializers/ThingDeserializer.cs

```diff
@@ -144,5 +189,87 @@ public ThingDeserializer(IHealthVaultConnection connection)
 
             return result;
         }
+
+        public ThingBase DeserializeItem(XPathNavigator thingNav)
+        {
+            ThingBase result;
+            Guid typeId = new Guid(thingNav.SelectSingleNode("type-id").Value);
```

Comment:
this is internal method .. there is a guarantee from the   internal users that this won't be null ..  

---
In reply to: [123101596](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/62#discussion_r123101596) [](ancestors = 123101596)